README.md

```diff
@@ -7,12 +7,20 @@ https://github.com/fxwiegand/terrarium-reverb/
 
 1. Install toolchain
 
+Linux
+
+```bash
+sudo apt install build-essential make cmake gcc-arm-none-eabi binutils-arm-none-eabi dfu-util
```

Comment:
I didn't actually test this, there might be more packages needed

Reply:
I think that this is everything! Hopefully someone other than us will use this and find an issue here lol.

Only potential issue is that the arm gcc that gets installed from apt-get is pretty old (2021). To install the latest version you need to go to [the arm developer website](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads)